### DISCUSSION TITLE
v0.9.0

### DISCUSSION THREAD

**Discussion by Mimetis** [Mimetis/Dotmim.Sync#631] [General]
`v0.9.0` has been released today
You can grab the packages here : https://www.nuget.org/packages?q=Dotmim.Sync

## Upgrade to version v0.9.0

Do not forget to call the `UpgradeAsync `method to upgrade your current schema version the last version.

- **Client Side** : No need to call `UpgradeAsync`, since it's handled by default by `DMS`
- **Server Side** : You need to create a `RemoteOrchestrator` with the correct `SyncSetup` and call the `UpgradeAsync` method

``` cs
// Upgrade to last version
if (await remoteOrchestrator.NeedsToUpgradeAsync())
   await remoteOrchestrator.UpgradeAsync(progress:progress);
```
When making the upgrade of your system (server / clients), be sure to remember that:
- A client upgraded **is not compatible** with a non upgraded server.
- A server upgraded **will be compatible** with non upgraded clients

That being said, be sure to firstly upgrade your server to the last version, call the `UpgradeAsync` (as explained before), then upgrade your clients.

Once upgraded your schema should be updated to the last version

## .Net 6.0

This version brings the support of `.Net 6.0` !

## Breaking Changes

The version `v0.9.0` is just a rebranding of the version `v0.8.1` (except the `.Net 6.0` support )
You will find all the changes for this version on the #602 discussion

**Comment by matevzcesar** [comment]
Big-ups for .net 6.0 support!
